test/src/routers/handlers/emailRouter.unit.ts

```diff
@@ -1,64 +1,172 @@
 import app from "../../../../src/app";
-import {EMAIL_CHECK_ANSWER_URL} from "../../../../src/config";
+import {
+    EMAIL_CHANGE_EMAIL_ADDRESS_URL,
+    EMAIL_CHECK_ANSWER_URL,
+    EMAIL_UPDATE_SUBMITTED_URL
+} from "../../../../src/config";
 import {HttpResponse} from "@companieshouse/api-sdk-node/dist/http/http-client";
 import {StatusCodes} from "http-status-codes";
 import {CheckAnswerHandler} from "../../../../src/routers/handlers/email/checkAnswer";
+import {ChangeEmailAddressHandler} from "../../../../src/routers/handlers/email/changeEmailAddress";
+import {UpdateSubmittedHandler} from "../../../../src/routers/handlers/email/updateSubmitted";
 
-const createdResponse: HttpResponse = {status: StatusCodes.CREATED};
-const noContentResponse: HttpResponse = {status: StatusCodes.NO_CONTENT};
 const okResponse: HttpResponse = {status: StatusCodes.OK};
+const movedTemporarilyResponse: HttpResponse = {status: StatusCodes.MOVED_TEMPORARILY};
 
 const clone = (objectToClone: any): any => {
-  return JSON.parse(JSON.stringify(objectToClone));
+    return JSON.parse(JSON.stringify(objectToClone));
 };
 
-describe("Confirm email router tests", () => {
-  const PAGE_HEADING = "Update a registered email address";
-
-  beforeEach(() => {
-    jest.clearAllMocks();
-  });
-
-  it("Should navigate to confirm email page", async () => {
-    const getSpy = jest.spyOn(CheckAnswerHandler.prototype, 'get').mockResolvedValue(okResponse);
-
-    await request(app)
-      .get(EMAIL_CHECK_ANSWER_URL)
-      .then((response) => {
-        expect(response.text).toContain(PAGE_HEADING);
-        expect(response.status).toBe(200);
-        expect(getSpy).toHaveBeenCalled();
-        expect(mocks.mockAuthenticationMiddleware).toHaveBeenCalled();
-      });
-  });
-
-  it("Should re-display check answer page when Email not confirmed", async () => {
-    await request(app)
-      .post(EMAIL_CHECK_ANSWER_URL)
-      .then((response) => {
-        expect(response.text).toContain("You need to accept the registered email address statement");
-        expect(response.status).toBe(200);
-        expect(mocks.mockAuthenticationMiddleware).toHaveBeenCalled();
-      });
-  });
-
-  it("Should navigate to update submitted page when Email confirmed", async () => {
-    const createSpy = jest.spyOn(rea, 'createRegisteredEmailAddressResource').mockResolvedValue(clone(createdResponse));
-    const transSpy = jest.spyOn(transactions, 'closeTransaction').mockResolvedValue(clone(noContentResponse));
-
-    await request(app)
-      .post(EMAIL_CHECK_ANSWER_URL)
-      .send({emailConfirmation: 'anything'})
-      .then((response) => {
-        expect(response.text).toContain("Registered email address update submitted");
-        expect(response.status).toBe(200);
-        expect(createSpy).toHaveBeenCalled();
-        expect(transSpy).toHaveBeenCalled();
-      });
-  });
+describe("Email router tests", () => {
+    const COMMON_PAGE_HEADING = "Update a registered email address";
```

Comment:
Do we need 2 separate constants both set to `Update a registered email address`?

Reply:
Good point Mike, was trying to keep test code separate but happy to replace all literals with constants if that's the way?

Reply:
done